### DISCUSSION TITLE
Ability to set the secondary input to FPP Remote Mode using xlights.

### DISCUSSION THREAD

**Discussion by onewithhammer** [forkineye/ESPixelStick#503] [General]
It would be great to have a way to set the FPP Remote Mode in xLights.

I working on a tutorial video using ESPixelSticks in FPP remote mode.  I'm able to use xLights to setup all configuration of EXPixelSticks (using DDP) with the exception of setting the Secondary Input to FPP Remote Mode.  The Secondary Input needs to be set using the ESPixelStick UI.

I'm not having any issues when starting with a fresh upload of v4-beta4 then using xLights to config the ESPixelSticks.  I have been testing my demo for 3 days now with ESPixelSticks running in FPP Remote mode and FPP Multisync.  I haven't seen a single reboot from 3 ESPixelSticks running as long as I don't use the UI.

**Comment by MartinMueller2003** [comment]
This is not under our control. You would need to ask the xLights team to add this. The ESPixelStick has no knowledge of xLights. xLights uses our APIs to perform operations. They would have to change the secondary input mode to the FPP Remote setting via a config download command.
